card/mod/01_history/set/all/actions.rb

```diff
@@ -0,0 +1,47 @@
+# -*- encoding : utf-8 -*-
```

Comment:
That's sort of what I was thinking with my request.  Dependency handling is a lot more complex than just configuring a load order for the core mods, and that would be an opportunity to leave specific items out of the list.  That might be enough for the standard modules.  Maybe you could just have a single array that had all of them, core or deck looking first in the deck, then the gem.

The full solution probably needs for the use cases to develop a bit so that we really need a more complex solution for dependencies.